### ISSUE TITLE
[Preview2] Microsoft.Windows.ApplicationModel.WindowsAppSDK.xml is incorrectly named

### ISSUE THREAD

**Issue by MarkIngramUK** [microsoft/WindowsAppSDK#1588]
Internal bug: https://task.ms/36701833

In the `microsoft.windowsappsdk.1.0.0-preview2` package, there are the following 2 files:

```
lib\uap10.0\Microsoft.Windows.ApplicationModel.WindowsAppRuntime.winmd
lib\uap10.0\Microsoft.Windows.ApplicationModel.WindowsAppSDK.xml
```

The XML file appears to be related to the winmd file, judging by it's contents:
```
<?xml version="1.0" encoding="utf-8"?>
<doc>
  <assembly>
    <name>Microsoft.Windows.ApplicationModel.WindowsAppRuntime</name>
  </assembly>
  <members>
  </members>
</doc>

```

So I'm assuming that `Microsoft.Windows.ApplicationModel.WindowsAppSDK.xml` actually needs to be called `Microsoft.Windows.ApplicationModel.WindowsAppRuntime.xml`.

If my assumption is wrong, apologies, feel free to close.

**Comment by andrewleader** [comment]
Updating milestone to 1.0.1, this did not make the GA but we'll fix it in servicing.

**Comment by bpulliam** [comment]
This was fixed in 1.1
